exp/internal/controllers/suite_test.go

```diff
@@ -31,8 +32,9 @@ import (
 )
 
 var (
-	env *envtest.Environment
-	ctx = ctrl.SetupSignalHandler()
+	timeout = time.Second * 30
```

Comment:
Where is `timeout` being used? I don't see it referenced in this file.

Reply:
It's in the machine_pool_controller_phases_test.go file, I'll move it there instead.